lesson_6/filter_func.py

```diff
@@ -0,0 +1,19 @@
+names = ('name_1', 'Yan', 'Yana', 'Dima', 'some_name')
+
+
+def len_higher_4(x):
+    if len(x) > 4:
```

Comment:
we can just return bool var without return value.. another word we can use just `return len(x) > 4` instead of condition here.. it will be more readable